src/app/app.module.ts

```diff
@@ -1,28 +1,25 @@
-/* ::START::LICENCE::
-Copyright eQualityTime ©2018, ©2019, ©2020, ©2021
-This file is part of OVFPlayer.
-OVFPlayer is free software: you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation, either version 3 of the License, or
-(at your option) any later version.
-OVFPlayer is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
-You should have received a copy of the GNU General Public License
-along with OVFPlayer.  If not, see <https://www.gnu.org/licenses/>.
-::END::LICENCE:: */
-import 'hammerjs';
```

Comment:
We've lost our license header.
Also we're no longer importing hammerjs, do we no longer need it? or did we never need it? (it looks like it's a js lib for gesture support :thinking: )

Reply:
I've added the licence back to this file, and updated the licence text and run the licence updater (which has therefore changed all the files ever, sorry)

Reply:
This file was replaced by the angular upgrade tool... so I don't really know about hammerjs... I haven't noticed any missing behaviour when testing it...

Reply:
(I'm wondering if I should roll back that last commit, just add the licence to that file, and do the licence update in a separate PR?)